### ISSUE TITLE
Is the loss function correct?

### ISSUE THREAD

**Issue by Alleycat404** [alexandru-dinu/cae#31]
Thanks for sharing. But I believe the loss function used in the paper "Lossy Image Compression with Compressive Autoencoders" is supposed to be Rate-distortion, why is MSE loss used instead?

**Comment by alexandru-dinu** [comment]
Hi,

I started with a simple implementation -- did not have the goal to fully reproduce the paper.

This is also mentioned in the [wiki](https://github.com/alexandru-dinu/cae/wiki):
> The models are inspired by [1], [2]. They are just experiments I have tried and do not fully implement the architectures described in the referenced papers.
[1] [Lossy Image Compression with Compressive Autoencoders, Theis et al.](https://arxiv.org/abs/1703.00395)
[2] [Variable Rate Image Compression with Recurrent Neural Networks, Toderici et al.](http://arxiv.org/abs/1511.06085)

**Comment by Alleycat404** [comment]
Thanks for your timely reply

**Comment by Alleycat404** [comment]
I see, thanks for your timely reply
